app/validation/validator.py

```diff
@@ -151,6 +153,75 @@ def validate_range_types_from_answers(self, json_to_validate):
 
         return errors
 
+    def _validate_min_max_ranges(self, json_to_validate):
+        errors = []
+        answers = {}
+
+        for block in self._get_blocks(json_to_validate):
+            for answer in self._get_answers_for_block(block):
+                answers[answer.get('id')] = answer
+
+        for answer_id, answer in answers.items():
+            referenced_answer_id = self._get_referenced_answer_id(answer)
+
+            if referenced_answer_id:
+                answer_min_max = self._get_range_for_answer(answer, answers)
+                # referenced_answer_range = self._get_range_for_answer(answers[referenced_answer_id], answers)
+                answer_range = answer_min_max[1] - answer_min_max[0]
+                if answer_range <= 0:
+                    error_message = 'The range of {} is outside the range of {}'.format(answer_id, referenced_answer_id)
+                    errors.append(self._error_message(error_message))
+
+        return errors
+
+    def _get_range_for_answer(self, answer, answers):
+        maximum = answer.get('max_value')
+        minimum = answer.get('min_value')
+
+        max_value = MAX_NUMBER
+        min_value = 0
+
+        if maximum:
+            max_value = self._get_maximum_value(answer, answers, min_value)
+        if minimum:
+            min_value = self._get_minimum_value(answer, answers, max_value)
+
+        return min_value, max_value
+
+    def _get_maximum_value(self, answer, answers, min_value):
```

Comment:
I don't think the naming of this method is quite right. If `value` is set, it returns the maximum possible value for the answer. If `answer_id` is set the maximum value becomes a range (the min to max of the referenced answer), and it returns the minimum possible value from that range. 

Also, min_value doesn't appear to be used.